addons/crm/data/crm_team_demo.xml

```diff
@@ -2,7 +2,7 @@
 <odoo>
     <data noupdate="1">
         <record id="sales_team.team_sales_department" model="crm.team" forcecreate="False">
-            <field name="assignment_domain">[['probability', '>=', '20']]</field>
+            <field name="assignment_domain">[['probability', '>=', 20]]</field>
```

Comment:
Feel free to make a separate PR for this so we can review it separately from the rest and merge it immediately if applicable.

Reply:
Here is the separate PR for the fix:
(master)  #191100 
(17.0)  #191098